scipy-1.0/paper.tex

```diff
@@ -848,22 +848,18 @@ \section*{Key technical improvements}
 
 \subsection*{Data structures}
 
-\subsubsection*{cKDTree}
-
-\input{ckdtree}
-
 \subsubsection*{Sparse matrices}
 
 \texttt{scipy.sparse} offers seven sparse matrix data structures,
 also known as sparse formats. The most important ones are the row- 
 and column-compressed formats (CSR and CSC, respectively). 
 These offer fast major-axis indexing and fast matrix-vector multiplication,
-and are used heavily in scikit-learn.
+and are used heavily throughout SciPy and dependent packages.
```

Comment:
Why only mention scikit-learn? 
Can we cite a fact about a different package without citation?

Reply:
The general statement you made is probably fine -- I'll just mark this as resolved with your adjustment